interfaces/Portalicious/src/app/components/form/form.component.ts

```diff
@@ -10,13 +10,13 @@ import { CreateMutationResult } from '@tanstack/angular-query-experimental';
  * - `FormSidebarComponent`
  */
 @Component({
-  selector: 'app-form-base-do-not-use',
+  selector: 'app-abstract-form-do-not-use',
```

Comment:
You couldn't resist to also include a "naming things"-code change, next the all the "invalidating cache"-code changes, right? :D

Reply:
![image](https://github.com/user-attachments/assets/d559c963-d77a-4da1-b4d8-7c7c699167b2)